HTTPlayground/main.py

```diff
@@ -0,0 +1,39 @@
+from http.server import HTTPServer, BaseHTTPRequestHandler
+from HTTPlayground.views import not_found
+from HTTPlayground.urls import urls
+from HTTPlayground.settings import URL, PORT
+from HTTPlayground.base import FileReader, send_headers
+
+
+class SimpleRequestHandler(BaseHTTPRequestHandler):
+
+    def do_GET(self):
+        if self.path == '/favicon.ico':
+            return
+        elif self.path.endswith(('.css', '.js')):
+            filename = self.path.rsplit('/', 1)[-1]
+            extension = filename.rsplit('.', 1)[-1]
+            content_type_part = "application" if extension == "js" else "text"
+            send_headers(self, content_type=f'{content_type_part}/{extension}')
+            self.wfile.write(FileReader.read(filename, mode='b'))
+        else:
+            body_content = f'{urls.get(self.path, not_found)(self)}'
```

Comment:
minor: why do we need to wrap view result in an f-string? Even if you want to make sure it's a string, it's better to use `str()`. F-strings are better for some templates like `f'the result is: {result}'`.

Reply:
![image](https://user-images.githubusercontent.com/37510854/63032353-54f64480-bebe-11e9-9537-0903c4632605.png)

Just for fun) And because it's more readable)

> minor: why do we need to wrap view result in an f-string? Even if you want to make sure it's a string, it's better to use `str()`. F-strings are better for some templates like `f'the result is: {result}'`.

Reply:
1. I'd argue which one is more explicit. At least it's not what f-strings are intended for. 
1. timing difference is negligible in the current scenario.
1. it's minor after all, so you can leave it as you prefer.